presto-product-tests/conf/docker/common/jdbc_db.yml

```diff
@@ -24,3 +24,12 @@ services:
       MYSQL_PASSWORD: swarm
       MYSQL_ROOT_PASSWORD: swarm
       MYSQL_DATABASE: test
+
+  sqlserver:
+    hostname: sqlserver
+    image: 'microsoft/mssql-server-linux'
+    ports:
+      - '1433:1433'
```

Comment:
Please use not default port if possible, similarly to what we have for postgres and mysql.
This is to prevent clash with dev env while working with another instance of MsSQL.